vllm/worker/cache_engine.py

```diff
@@ -48,6 +48,9 @@ def __init__(
         # Initialize the cache.
         self.gpu_cache = self._allocate_kv_cache(self.num_gpu_blocks, "cuda")
         self.cpu_cache = self._allocate_kv_cache(self.num_cpu_blocks, "cpu")
+        # Zero out the first block in the cache, in case it gets used as
+        # 'null_block' in the CpuGpuBlockAllocator
+        self.attn_backend.zero_block(self.gpu_cache, 0)
```

Comment:
Can we get rid of this and the attn_backend interface changes if we zero-out the entire cache from the start?

Reply:
asking internally for a vibe check